app/services/moves/finder.rb

```diff
@@ -69,12 +70,22 @@ def apply_filter(scope, param_name)
     def apply_date_range_filters(scope)
       scope = scope.where('date >= ?', filter_params[:date_from]) if filter_params.key?(:date_from)
       scope = scope.where('date <= ?', filter_params[:date_to]) if filter_params.key?(:date_to)
-      # created_at is a date/time, so inclusive filtering has to be subtlely different
+      # created_at is a date/time, so inclusive filtering has to be subtly different
       scope = scope.where('moves.created_at >= ?', filter_params[:created_at_from]) if filter_params.key?(:created_at_from)
       scope = scope.where('moves.created_at < ?', Date.parse(filter_params[:created_at_to]) + 1) if filter_params.key?(:created_at_to)
       scope
     end
 
+    def apply_date_of_birth_filters(scope)
+      return scope unless filter_params.key?(:date_of_birth_from) || filter_params.key?(:date_of_birth_to)
+
+      # only join on person if necessary, otherwise moves without people are not included
+      scope = scope.joins(:person)
+      scope = scope.where('people.date_of_birth >= ?', filter_params[:date_of_birth_from]) if filter_params.key?(:date_of_birth_from)
```

Comment:
what do you think about indexing `date_of_birth`?

Reply:
Index added.